server/app.js

```diff
@@ -259,9 +259,33 @@ app.get('/print/:id', asyncHandler(async (req, res, next)=>{
 	return next();
 }));
 
+const nodeEnv = config.get('node_env');
+const isLocalEnvironment = config.get('local_environments').includes(nodeEnv);
+// Login
+app.post('/login', (req, res)=>{

```

Comment:
What do you think about renaming this route to `/localLogin` or something?

Reply:
I've just been experimenting with renaming the route `/local/login` and wrapping it a `isLocalEnvironment` check - so the route shouldn't even get loaded in Production.

Reply:
Pushed the commit now